multisig/proposals/vip-006/vip-006-sepolia.ts

```diff
@@ -2,19 +2,19 @@ import { parseUnits } from "ethers/lib/utils";
 
 import { makeProposal } from "../../../src/utils";
 
-export const REWARD_DISTRIBUTOR_CORE_0 = "0xEA3bb4A1C6218E31e435F3c23E0E9a05A40B7F40";
-export const REWARD_DISTRIBUTOR_CORE_1 = "0xAbBAe88E3E62D6Ffb23D084bDFD2A1Dc45e15879";
-export const REWARD_DISTRIBUTOR_CORE_2 = "0x6B3f9BBbBC8595f9c3C8e0082E95C45F98239E1b";
-export const REWARD_DISTRIBUTOR_CORE_3 = "0x5e128F6B554589D3B1E91D53Aee161A70F439062";
-const COMPTROLLER_CORE = "0x7Aa39ab4BcA897F403425C9C6FDbd0f882Be0D70";
-const XVS = "0xDb633C11D3F9E6B8D17aC2c972C9e3B05DA59bF9";
+export const REWARD_DISTRIBUTOR_CORE_0 = "0x78d1D10F4772982B82497cA010aBFF0601B66c4a";
+export const REWARD_DISTRIBUTOR_CORE_1 = "0x3cE1A4C37C6F5106F3a04B93515Ce53F26C8DAC2";
+export const REWARD_DISTRIBUTOR_CORE_2 = "0xf06f1A32C3b5504E7a932d736F2d10612abC55d3";
+export const REWARD_DISTRIBUTOR_CORE_3 = "0xa822b4aadc53A7641ce84c690C3109fD336440Aa";
```

Comment:
In these contracts, the pending owner is not the guardian contract. The simulation fails